### PR TITLE
Add vcpkg installation instructions

### PR DESCRIPTION
**Pull request by PhoebeHui**
Dlib is available as a port in VCPKG , documenting the install process here will help users get started by providing a single set of commands to build dlib, ready to be included in their projects.

VCPKG is a C++ library manager that simplifies installation for dlib and other project dependencies, we also test whether our library ports build in various configurations (dynamic, static) on various platforms (OSX, Linux, Windows: x86, x64, UWP, ARM) to keep a wide coverage for users.

I'm a maintainer for vcpkg, and here is what the port script looks like. We try to keep the library maintained as close as possible to the original library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ mkdir build; cd build; cmake .. -DUSE_AVX_INSTRUCTIONS=1; cmake --build .
 
 Doing so will make some things run faster.
 
+Alternatively, if you are using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager you can download and install dlib with CMake integration in a single command:
+```bash
+vcpkg install dlib
+```
+
```
Comment (davisking) [inline README.md]: Thanks for the PR.  Isn't the next paragraph somewhat contrary to this?  If someone is using vcpkg they are almost certainly using visual studio correct?  They would then not run cmake as the next paragraph indicates?  This should be clarified and this new text needs to be coherent with the surrounding text. At least by making it clear that this is an either or thing.  Either cmake, or alternatively vcpkg when using visual studio.

Comment (PhoebeHui) [reply README.md]: @davisking, thanks for your reviewing!

VCPKG support to build dlib on Windows, Linux ans OSX. On windows, it use visual studio to build dlib, so if someone using vcpkg, it's not only using visual studio, but also gcc or clang, I considered to add it separately, could you give me some guidelines that where should I add the instructions?

**Comment by davisking** [comment]
I moved the text down a little to a more appropriate place, and it's live on https://github.com/davisking/dlib now :)

**Comment by PhoebeHui** [comment]
@davisking, thanks for your help!

**Pull request closed without merging**